src/firefly/js/visualize/ui/ImageSelectPanel.jsx

```diff
@@ -50,27 +51,40 @@ export const keyMap = {
     'fitslist':    'SELECTIMAGEPANEL_FITS_list',
     'fitsextinput':'SELECTIMAGEPANEL_FITS_extinput',
     'fitsupload':  'SELECTIMAGEPANEL_FITS_upload',
-    'radiusfield': 'SELECTIMAGEPANEL_ImgFeature_radius',
+    'sizefield': 'SELECTIMAGEPANEL_ImgFeature_radius',
```

Comment:
just in case the key is not named as a JavaScript identifier. It is actually not needed here.